io.github.dummerle.rare.json

```
@@ -0,0 +1,48 @@
+{
+    "app-id": "io.github.dummerle.rare",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "rare",
+    "rename-icon": "rare",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=home",
+        "--device=all",
+        "--talk-name=org.freedesktop.Flatpak"
+    ],
+    "modules": [
+        {
+            "name": "desktop-entry",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -D rare.desktop /app/share/applications/io.github.dummerle.rare.desktop"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "path": "https://raw.githubusercontent.com/Dummerle/Rare/1.8.9/misc/rare.desktop"
```

```suggestion
                    "url": "https://raw.githubusercontent.com/Dummerle/Rare/1.8.9/misc/rare.desktop"
```

and you need a sha256